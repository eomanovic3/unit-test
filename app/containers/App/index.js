/**
 *
 * App
 *
 * This component is the skeleton around the actual pages, and should only
 * contain code that should be seen on all pages. (e.g. navigation bar)
 */

import React from 'react';
import styled from 'styled-components';
import 'bootstrap/dist/css/bootstrap.min.css';
import history from 'utils/history';
import { Helmet } from 'react-helmet';

import HomePage from 'containers/HomePage/Loadable';
import DetailPage from 'containers/DetailPage/Loadable';
import VideoPlayer from 'containers/VideoPlayer/Loadable';
import SearchPage from 'containers/SearchPage/Loadable';
import { Route, Switch } from 'react-router-dom';
import GlobalStyle from '../../global-styles';

const AppWrapper = styled.div`
  margin: 0 auto;
  display: flex;
  min-height: 100%;
  flex-direction: column;
`;

export default function App() {
  return (
    <AppWrapper>
      <Helmet
        titleTemplate="%s - React.js Boilerplate"
        defaultTitle="React.js Boilerplate"
      >
        <meta name="description" content="A React.js Boilerplate application" />
      </Helmet>
      <Switch>
        <Route exact path="/" component={HomePage} history={history} />
        <Route path="/features" component={HomePage} history={history} />
        <Route path="/search" component={SearchPage} history={history} />
        <Route
          path="/detailPage/:movieType/:videoId"
          component={DetailPage}
          history={history}
        />
        <Route
          path="/videoPlayer/:videoId"
          component={VideoPlayer}
          history={history}
        />
      </Switch>
      <GlobalStyle />
    </AppWrapper>
  );
}
