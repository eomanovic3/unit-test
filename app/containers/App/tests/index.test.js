// import { CountryList } from '../index';

import React from 'react';
import Adapter from 'enzyme-adapter-react-16/build';
import Enzyme, { shallow } from 'enzyme';
import App from '../index';
Enzyme.configure({ adapter: new Adapter() });

describe('<App />', () => {
  it('should render with videos', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
