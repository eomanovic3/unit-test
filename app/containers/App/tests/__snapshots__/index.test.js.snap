// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render with videos 1`] = `ShallowWrapper {}`;
